src/main/java/at/hannibal2/skyhanni/features/garden/contest/OpenContestInElitebotDev.kt

```diff
@@ -0,0 +1,95 @@
+package at.hannibal2.skyhanni.features.garden.contest
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.cleanName
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.KeyboardManager.isKeyHeld
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NumberUtil.formatNumber
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.SimpleTimeMark.Companion.asTimeMark
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import net.minecraft.client.gui.inventory.GuiChest
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object OpenContestInElitebotDev {
+
+    private val config get() = SkyHanniMod.feature.garden.eliteWebsite
+
+    private val EARLIEST_CONTEST: SimpleTimeMark = SkyBlockTime(year = 100, month = 6, day = 18).asTimeMark()
+
+    private const val ELITEBOT_DOMAIN: String = "https://elitebot.dev"
+    private const val ELITEBOT_CONTESTS: String = "$ELITEBOT_DOMAIN/contests"
+    private const val ELITEBOT_UPCOMING: String = "$ELITEBOT_CONTESTS/upcoming"
+    private const val ELITEBOT_RECORDS_SUFFIX: String = "records"
+
+    private val elitebotDevRepoGroup = RepoPattern.group("garden.contest.elitebot")
+    private val calendarDatePattern by elitebotDevRepoGroup.pattern(
+        "date.chestanditem",
+        "(?<sbTime>(?<month>(?:Early |Late )?(?:Winter|Spring|Summer|Autumn|Fall))(?: (?<date>\\d+)(?:nd|rd|th|st))?, Year (?<year>[\\d,.]+))"
```

Comment:
this feels duplicate to another feature of yours. maybe write a common api or use existing data instead.